jwst/jump/jump_step.py

```diff
@@ -22,6 +22,10 @@ class JumpStep(Step):
         flag_4_neighbors = boolean(default=True) # flag the four perpendicular neighbors of each CR
         max_jump_to_flag_neighbors = float(default=1000) # maximum jump sigma that will trigger neighbor flagging
         min_jump_to_flag_neighbors = float(default=10) # minimum jump sigma that will trigger neighbor flagging
+        after_jump_flag_dn1 = float(default=0) # 1st flag groups after jump above DN threshold
+        after_jump_flag_time1 = float(default=0) # 1st flag groups after jump groups within specified time
```

Comment:
I'm wondering about how the default case (everything set to zero) ends up getting handled in the lower-level functions. If the time arguments are zero, that will equate to 0 groups getting flagged, but a DN level of zero (which gets converted to zero electrons) will cause all jumps to be above the threshold and hence a whole lot of pixels could end up getting calculations done on them only to find that 0 groups get flagged for them. Is there something down in the lower-level code that checks for a threshold of zero before bothering to do the calculations?

Reply:
i was also wondering this. i think a check in stcal for all 0 values should be done to skip this, and make it clear that its being skipped

Reply:
This is done in stcal.  If time[1,2] is zero, then skip that after flagging.

Reply:
To be clear if n[1,2] is zero this happens.  As time[1,2] -> n[1,2] via the group time.

Reply:
There are tests for this behavior in stcal.